relayer/chains/substrate/event_parser.go

```diff
@@ -244,26 +259,53 @@ func (pkt *packetInfo) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 func (pkt *packetInfo) parseAttrs(log *zap.Logger, attributes interface{}) {
 	attrs := attributes.(ibcEventQueryItem)
 
-	pkt.Sequence = attrs["sequence"].(uint64)
+	var err error
+
+	var height exported.Height
+	if height, err = parseHeight(attrs["height"]); err != nil {
+		log.Error("error parsing connection height: ",
+			zap.Error(err),
+		)
+		return
+	}
+	pkt.Height = height.GetRevisionHeight()
+
+	pkt.Sequence = cast.ToUint64(attrs["sequence"].(float64))
```

Comment:
See other comment regarding type assertion to `float64` and then casting to `uint64`